### ISSUE TITLE
Scatter circle disappear when chart scaled

### ISSUE THREAD

**Issue by kou0117** [danielgindi/Charts#3185]
if you have more than 2 scatter on the chart and scale the chart to the extent that any scatter get invisible, all scatters disappear.

![scatter disappeared](https://user-images.githubusercontent.com/26572756/34977050-7205d22c-fadc-11e7-8515-3143214ad0de.gif)

if you know the solution, please tell me.
Thank you.

**Comment by liuxuan30** [comment]
can this be reproduced with ChartsDemo? It's a bug and should be fixed.

**Comment by kou0117** [comment]
Thank you for your reply but I got fixed.
when I appended ChartDataEntry In order of new, scatter circle disappeared when chart scaled.
However, when I appended ChartDataEntry In order of oldest, scatter circle didn't disappear.

I assume that when the scatter appended first get invisible, every scatter also get invisible. 

In the case of my GIF attached above, I appended the left scatter first.
Thank you.
